apps/api/src/hook/processors/webhook.processor.ts

```diff
@@ -54,13 +56,27 @@ export class HookService {
       console.log('Webhook Request Detail:', axiosConfig);
       console.log('Webhook Response Detail:', result.status, result.data);
 
-      return {
+      const hookResult = {
         response: {
           status: result.status,
           headers: result.headers,
           data: result.data,
         },
-      };
+        success: result.status === 200,
+      } as WebHookResponse;
+
+      if (success_path) {
+        const navigationPath = success_path.substring(2);
```

Comment:
It is better to do it by looking at the length rather than fixed. like this
const navigationPath = success_path.substring(Constants.prefixPattern.length);